src/main/java/de/tum/in/www1/exerciseapp/web/rest/QuizSubmissionResource.java

```diff
@@ -27,8 +35,51 @@
     private static final String ENTITY_NAME = "quizSubmission";
 
     private final QuizSubmissionRepository quizSubmissionRepository;
-    public QuizSubmissionResource(QuizSubmissionRepository quizSubmissionRepository) {
+    private final QuizExerciseRepository quizExerciseRepository;
+    private final ResultRepository resultRepository;
+    private final ParticipationService participationService;
+
+    public QuizSubmissionResource(QuizSubmissionRepository quizSubmissionRepository, QuizExerciseRepository quizExerciseRepository, ResultRepository resultRepository, ParticipationService participationService) {
         this.quizSubmissionRepository = quizSubmissionRepository;
+        this.quizExerciseRepository = quizExerciseRepository;
+        this.resultRepository = resultRepository;
+        this.participationService = participationService;
+    }
+
+    /**
+     * GET  /courses/{courseId}/exercises/{exerciseId}/submissions/my-latest : Get the latest quizSubmission for the given course.
+     *
```

Comment:
I don't really get what this call is used for. Is it that the user can get an "open" submission, in case he has lost his connection and reconnects?
Please document it properly.